ghost/core/core/frontend/helpers/content.js

```diff
@@ -19,7 +19,8 @@ function restrictedCta(options) {
     options = options || {};
     options.data = options.data || {};
     _.merge(this, {
-        accentColor: (options.data.site && options.data.site.accent_color)
+        accentColor: (options.data.site && options.data.site.accent_color),
```

Comment:
Although we can't remove this line with the given changes, we should add a comment above it something like: `// @deprecated in Ghost 5.16.1 never documented & removed from core templates`